### PR TITLE
fix twitter preview

### PR DESCRIPTION
**Pull request by johngruen**
@pdehaan i think we'll have to merge this to see if it works in dev #910

### CODE REVIEW

File: testpilot/frontend/templates/testpilot/frontend/index.html [hunk 1]
```diff
@@ -16,21 +16,24 @@
...
-        content="{{ request.build_absolute_uri(static('images/thumbnail-facebook.png')) }}" />
-    <meta property="twitter:image"
-        content="{{ request.build_absolute_uri(static('images/thumbnail-twitter.png')) }}" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:description" content="{{ meta_description }}" />
+    <meta name="twitter:image" content="{{ request.build_absolute_uri(static('images/thumbnail-twitter.png')) }}" />
+    <meta name="twitter:site" content="@FxTestPilot" />
+    <meta name="twitter:title" content="{{ meta_title }}" />
+
+
```
Comment (pdehaan) [inline testpilot/frontend/templates/testpilot/frontend/index.html:36]: /swoon Sorted by type then sub-sorted by meta name?

Comment (johngruen) [inline testpilot/frontend/templates/testpilot/frontend/index.html:36]: your issue was my inspiration.

**Comment by pdehaan** [comment]
**PSA:** We currently don't have **gulp-imagemin** running on this repo (ref #908), so make sure we're checking in optimized assets.

**Comment by johngruen** [comment]
Also fixes #909

**Comment by pdehaan** [comment]
LGTM, and @ckprice gave it his seal of approval (so we can blame him if it goes wrong).

**Pull request merged**